src/Explorer/Panes/AddGlobalSecondaryIndexPanel/AddGlobalSecondaryIndexPanel.tsx

```diff
@@ -0,0 +1,430 @@
+      } else {
+        offerThroughput = globalSecondaryIndexThroughput;
+      }
+    }
+
+    const createGlobalSecondaryIndexParams: DataModels.CreateMaterializedViewsParams = {
+      materializedViewId: globalSecondaryIdTrimmed,
+      materializedViewDefinition: globalSecondaryIndexDefinition,
+      databaseId: selectedSourceContainer.databaseId,
+      databaseLevelThroughput: databaseLevelThroughput,
+      offerThroughput: offerThroughput,
+      autoPilotMaxThroughput: autoPilotMaxThroughput,
+      analyticalStorageTtl: getAnalyticalStorageTtl(),
+      indexingPolicy: indexingPolicy,
+      partitionKey: partitionKeyPaths,
+      uniqueKeyPolicy: uniqueKeyPolicy,
+      vectorEmbeddingPolicy: vectorEmbeddingPolicyFinal,
+      fullTextPolicy: fullTextPolicy,
+    };
+
+    setIsExecuting(true);
+
+    try {
+      await createGlobalSecondaryIndex(createGlobalSecondaryIndexParams);
+      await explorer.refreshAllDatabases();
+      TelemetryProcessor.traceSuccess(Action.CreateGlobalSecondaryIndex, telemetryData, startKey);
+      useSidePanel.getState().closeSidePanel();
+    } catch (error) {
+      const errorMessage: string = getErrorMessage(error);
+      setErrorMessage(errorMessage);
+      setShowErrorDetails(true);
+      const failureTelemetryData = { ...telemetryData, error: errorMessage, errorStack: getErrorStack(error) };
+      TelemetryProcessor.traceFailure(Action.CreateGlobalSecondaryIndex, failureTelemetryData, startKey);
+    } finally {
+      setIsExecuting(false);
+    }
+  };
+
+  return (
+    <form className="panelFormWrapper" id="panelGlobalSecondaryIndex" onSubmit={submit}>
+      {errorMessage && (
+        <PanelInfoErrorComponent message={errorMessage} messageType="error" showErrorDetails={showErrorDetails} />
+      )}
+      <div className="panelMainContent">
+        <Stack>
+          <Stack horizontal>
+            <span className="mandatoryStar">*&nbsp;</span>
+            <Text className="panelTextBold" variant="small">
+              Source container id
+            </Text>
+          </Stack>
+          <Dropdown
+            placeholder="Choose source container"
+            options={sourceContainerOptions}
+            defaultSelectedKey={selectedSourceContainer?.rid}
+            styles={chooseSourceContainerStyles()}
+            style={chooseSourceContainerStyle()}
+            onChange={(_, options: IDropdownOption) => setSelectedSourceContainer(options.data as Collection)}
+          />
+          <Separator className="panelSeparator" />
+          <Stack horizontal>
+            <span className="mandatoryStar">*&nbsp;</span>
+            <Text className="panelTextBold" variant="small">
+              Global secondary index container id
+            </Text>
+          </Stack>
+          <input
+            id="globalSecondaryIndexId"
+            type="text"
+            aria-required
+            required
+            autoComplete="off"
+            pattern="[^/?#\\]*[^/?# \\]"

```

Comment:
Update this to the changes I introduced with [this change](https://github.com/Azure/cosmos-explorer/commit/41f5401016bdbb3b6c8d4a8ddbf3f765257b3317) so that the input validation works.